devices/nue_3a.js

```diff
@@ -247,6 +273,40 @@ module.exports = [
         description: 'Smart 7W E27 light bulb',
         extend: extend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['LXN59-CS27LX1.0'],
+        model: 'ZW-EU-4C',
+        vendor: 'Nue / 3A',
+        description: 'ZigBee smart curtain switch',
+        fromZigbee: [fz.cover_position_tilt, fzLocal.LXN59_cover_state_via_onoff, fzLocal.LXN59_cover_options],
+        toZigbee: [tz.cover_state, tz.cover_position_tilt],
+        meta: {disableDefaultResponse: true},
+        exposes: [e.cover_position(), exposes.enum('moving', ea.STATE, ['UP', 'STOP', 'DOWN'])],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint1 = device.getEndpoint(1);
+            await reporting.bind(endpoint1, coordinatorEndpoint, ['genOnOff']);
+            await reporting.onOff(endpoint1);
+            await reporting.bind(endpoint1, coordinatorEndpoint, ['closuresWindowCovering']);
+            await reporting.currentPositionLiftPercentage(endpoint1);
+            device.powerSource = 'Mains (single phase)';
+            device.save();
+        },
+    },
+    {
+        zigbeeModel: ['LXX60-CS27LX1.0'],
+        model: 'ZW-EU-02',
+        vendor: 'Nue / 3A',
+        description: 'ZigBee smart curtain switch',
```

Comment:
@a-bailey is this really a curtain switch? Or is it the 1 channel switch from the Amazon page (?)